app/src/main/java/com/example/directory/models/Address.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.directory.models
+
+data class Address(
+    val city: String,
+    val coordinates: Coordinates,
+    val country: String,
+    val state: String,
+    val street_address: String,
+    val street_name: String,
+    val zip_code: String
```

Comment:
Variable name in kotlin should not have underscore. It can be ex. streetAddress
A better way to handle this is by serializing json with Google gson library https://stackoverflow.com/questions/44117970/kotlin-data-class-from-json-using-gson/44118828#44118828